e2e/interchaintestv8/testvalues/values.go

```diff
@@ -77,7 +77,7 @@ const (
 	// SP1ICS07FixturesDir is the directory where the SP1ICS07 fixtures are stored.
 	SP1ICS07FixturesDir = "test/sp1-ics07/fixtures"
 	// RustFixturesDir is the directory where the Rust fixtures are stored.
-	RustFixturesDir = "packages/ethereum-light-client/src/test_utils/fixtures"
+	RustFixturesDir = "packages/ethereum/ethereum-light-client/src/test_utils/fixtures"
```

Comment:
Eventually, I want to remove the `ethereum-` prefix on these packages's file names. Super easy change